File: Modul 4/C/PRAK404-2310817310010-AkmallullailSyaban.c
#include <stdio.h>

int main (){
    float program, a, b, hasil;
    while (1){
        printf("Pilih Program\n1.Penjumlahan\n2.Pengurangan\n3.Perkalian\n4.Pembagian\n5.Exit\n");
        printf("Masukkan Pilihan :");
        scanf("%f", &program);
        if (program < 1 || program > 5){
            printf("Input anda salah, silahkan coba lagi\n\n");
        } else if (program == 5){
            printf("Terimakasih, telah menggunakan kalkulator Akmallullail Sya'ban");
            break;
        }
        if (program == 1){
            printf("Masukkan Nilai Pertama :");
            scanf("%f", &a);
            printf("Masukkan Nilai Kedua :");
            scanf("%f", &b);
            hasil = a + b;
            printf("Hasil Penjumlahan antara %.2f dan %.2f adalah %.2f\n\n", a, b, hasil);}
        else if(program == 2){
            printf("Masukkan Nilai Pertama :");
            scanf("%f", &a);
            printf("Masukkan Nilai Kedua :");
            scanf("%f", &b);
            hasil = a - b;
            printf("Hasil Pengurangan antara %.2f dan %.2f adalah %.2f\n\n", a, b, hasil);}
        else if(program == 3){
            printf("Masukkan Nilai Pertama :");
            scanf("%f", &a);
            printf("Masukkan Nilai Kedua :");
            scanf("%f", &b);
            hasil = a * b;
            printf("Hasil Perkalian antara %.2f dan %.2f adalah %.2f\n\n", a, b, hasil);}
        else if(program == 4){
            printf("Masukkan Nilai Pertama :");
            scanf("%f", &a);
            printf("Masukkan Nilai Kedua :");
            scanf("%f", &b);
            hasil = a / b;
            printf("Hasil Pembagian antara %.2f dan %.2f adalah %.2f\n\n", a, b, hasil);}
    }
    return 0;
}

File: Modul 4/Python/PRAK404-2310817310010-AkmallullailSyaban.py
while True:
    print("Pilih Program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    program = int(input("Masukkan Pilihan: "))

    if program == 5:
        print("Terimakasih telah menggunakan kalkulator Akmallullail Sya'ban")
        break
    elif program == 1:
        a = float(input("Masukkan Nilai Pertama: "))
        b = float(input("Masukkan Nilai Kedua: "))
        hasil = a + b
        print(f"Hasil penjumlahan antara {a:.2f} dan {b:.2f} adalah {hasil:.2f}\n")
    elif program == 2:
        a = float(input("Masukkan Nilai Pertama: "))
        b = float(input("Masukkan Nilai Kedua: "))
        hasil = a - b
        print(f"Hasil pengurangan antara {a:.2f} dan {b:.2f} adalah {hasil:.2f}\n")
    elif program == 3:
        a = float(input("Masukkan Nilai Pertama: "))
        b = float(input("Masukkan Nilai Kedua: "))
        hasil = a * b
        print(f"Hasil perkalian antara {a:.2f} dan {b:.2f} adalah {hasil:.2f}\n")
    elif program == 4:
        a = float(input("Masukkan Nilai Pertama: "))
        b = float(input("Masukkan Nilai Kedua: "))
        hasil = a / b
        print(f"Hasil pembagian antara {a:.2f} dan {b:.2f} adalah {hasil:.2f}\n")
    else:
        print("Input anda salah, silahkan coba lagi\n")
        print("")